images/scorecard-untar/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8:8.7
+FROM registry.access.redhat.com/ubi8/ubi-micro:8.9
```

Comment:
does ubi-micro include the `tar` command? scorecard uses this image in container within a Pod for untaring bundle contents using the `tar` command. Without the `tar` command present in this image, I believe scorecard will always fail.